vault_core/src/vault.rs

```
@@ -178,9 +178,9 @@ impl Vault {
             vrt_ready_to_claim_amount: PodU64::from(0),
             last_fee_change_slot: PodU64::from(current_slot),
             last_full_state_update_slot: PodU64::from(current_slot),
-            deposit_fee_bps: PodU16::from(deposit_fee_bps),
-            withdrawal_fee_bps: PodU16::from(withdrawal_fee_bps),
-            reward_fee_bps: PodU16::from(reward_fee_bps),
+            deposit_fee_bps: PodU16::from(deposit_fee_bps.min(MAX_BPS)),
```

need to fail loudly if these are exceeded